build.cake

```diff
@@ -277,17 +277,14 @@ Task("BuildFx")
 ///  Packages projects specified in PackageProjects
 /// </summary>
 Task("DotnetPack")
-    .IsDependentOn("Cleanup")
-    .IsDependentOn("Setup")
-    .IsDependentOn("Restore")
```

Comment:
So these were removed because DotnetPack was moved out of the build step. The build step has Cleanup, Setup, Restore as pre-steps to be completed before DotnetPack can run. But when DotnetPack was moved out to go with DotnetPackNuspec, these dependencies had to be removed, else it was cleaning up everything that was built in the prior step.

Reply:
I would suggest also renaming this to DotnetPackNoBuild in that case to make it explicitly clear that this step doesn't do a normal dotnet pack (which would rebuild)